aeon/dj_pipeline/utils/streams_maker.py

```diff
@@ -135,7 +136,7 @@ def get_device_stream_template(device_type: str, stream_type: str, streams_modul
     for col in stream.columns:
         if col.startswith("_"):
             continue
-        table_definition += f"{col}: longblob\n\t\t\t"
+        table_definition += f"{col}: longblob\n    "
```

Comment:
I thought the `\t\t\t` is important to ensure the correct indentation in the code definition

Reply:
The text is indented weirdly with `\t\t\t` so I had to use space instead. Couldn't figure out why exactly. It's working though.